lib/fizzy/execute.cpp

```diff
@@ -79,6 +79,21 @@ std::vector<uint64_t> execute(const module& _module, funcidx _function, std::vec
             stack.push(a + b);
             break;
         }
+        case instr::i32_wrap_i64: {
+            const auto a = static_cast<uint64_t>(stack.pop());
+            stack.push(a & 0x00000000ffffffff);
```

Comment:
```suggestion
            stack.push(static_cast<uint32_t>(stack.pop()));
```

that's my preference.